libraries/src/CMS/Application/ApiApplication.php

```diff
@@ -134,5 +134,41 @@ public function getTemplate($params = false)
 		// The API application should not need to use a template
 		return 'system';
 	}
-}
 
+	/**
+	 * Route the application.
+	 *
+	 * Routing is the process of examining the request environment to determine which
+	 * component should receive the request. The component optional parameters
+	 * are then set in the request object to be processed when the application is being
+	 * dispatched.
+	 *
+	 * @return  void
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	protected function route()
+	{
+		$uri = \JUri::getInstance();
+		$router = static::getRouter();
+
+		// Trigger the onBeforeApiRoute event.
+		PluginHelper::importPlugin('system');
```

Comment:
Let's make this a 'new' plugin type called 'webservices' rather than using 'system'